### PR TITLE
Feature/editor

### PR DESCRIPTION
**Pull request by ghislaineguerin**
@ezequielc I'm adding all the staged links here

https://displays-stage-1.risevision.com
https://storage-stage-1.risevision.com
https://schedules-stage-1.risevision.com
https://store-stage-1.risevision.com/

### CODE REVIEW

**Comment by ghislaineguerin** [comment]
https://storage-stage-1.risevision.com/
https://displays-stage-1.risevision.com/

**Comment by ezequielc** [comment]
@ghislaineguerin 
https://help-stage.risevision.com/fix-update-common-styles/_site/user
https://developer-stage.risevision.com/apps/list
https://apps-stage-0.risevision.com/

File: src/ui-components/subscription.scss [hunk 1]
```diff
@@ -7,11 +7,16 @@
   padding: 6px 7px;
 
   text-align: center;
-  span
+  > span
```
Comment (RV-Adrian) [inline src/ui-components/subscription.scss]: @ghislaineguerin Tacked these on to your PR, thanks!

**Comment by ghislaineguerin** [comment]
@adriandd @ezequielc are we good to merge?

**Comment by ezequielc** [comment]
@ghislaineguerin Looks good to me, but I defer to @adriandd Cheers

**Comment by RV-Adrian** [comment]
@ghislaineguerin @ezequielc 
Looks like a number of changes covered, on a number of staged environments. Making me nervous. :) Is this risky to do on a Friday do you think?

**Comment by ezequielc** [comment]
@RV-Adrian yes, good point, we talked about it as well. We are going to update the apps on Monday. Thanks!

**Comment by RV-Adrian** [comment]
@ghislaineguerin @ezequielc ok cool! just that i've hit more than a few bugs today. It's made me paranoid. :) I'll continue to check the staged environments, thanks.

**Pull request merged**